File: src/main/tests/pythonGenerated/lambda.py
import typing

def main() -> int:
    add = (lambda a,b: a + b)
    print("add(5, 7) = ", add(5,7), "\n", sep="", end="")
    square = (lambda x: x * x)
    print("square(6) = ", square(6), "\n", sep="", end="")
    return 0

if __name__ == "__main__":
    main()

File: src/main/tests/input/lambda.cpp
#include <iostream>

int main() {
    auto add = [](int a, int b) { return a + b; };

    std::cout << "add(5, 7) = " << add(5, 7) << "\n";

    auto square = [](int x) { return x * x; };

    std::cout << "square(6) = " << square(6) << "\n";

    return 0;
}
